resources/carrier/ru/374.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Manually translated
+# Translated manually from en/374.txt
 # http://en.wikipedia.org/wiki/Telephone_numbers_in_Armenia#Mobile_network_codes
```

Comment:
Please delete line 16 with the Wikipedia link and add a period at the end of line 15.
See https://github.com/googlei18n/libphonenumber/blob/master/resources/geocoding/ru/374.txt - we want to be consistent with that.